crowbar_framework/app/models/deployer_service.rb

```diff
@@ -112,6 +112,28 @@ def transition(inst, name, state)
 
       node.save if save_it
 
+      role = RoleObject.find_role_by_name "deployer-config-#{inst}"
+      unless role.default_attributes["deployer"]["use_allocate"] and !node.admin?
```

Comment:
Style/AndOr: Use `&&` instead of `and`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)